nextcord/channel.py

```diff
@@ -990,6 +1037,27 @@ def last_message(self) -> Optional[Message]:
         """
         return self._state._get_message(self.last_message_id) if self.last_message_id else None
 
+    @property
+    def available_tags(self) -> List[ForumTag]:
+        """List[:class:`ForumTag`]: Returns all the tags available in this channel."""
+
+        return list(self._available_tags.values())
+
+    def get_tag(self, id: int, /) -> Optional[ForumTag]:
+        """Returns a tag from this channel by its ID.
```

Comment:
```suggestion
        """Returns a tag from this channel by its ID.
        
        .. versionadded:: 2.3
```

Reply:
The entirety of `ForumTag` is being added in 2.3, so I don't think this is necessary

Reply:
Well this method is in `ForumChannel`, not `ForumTag`.